### PR TITLE
DOC: Add details for running CreateNewExample.py

### PR DESCRIPTION
**Pull request by Leengit**
Explain what `example_name` and `class_name` are, as used in `CreateNewExample.py`.

### CODE REVIEW

**Comment by Leengit** [comment]
Please review for accuracy, formatting, English, and clarity.

File: Documentation/Contribute/ContributeWithGit.rst [hunk 1]
```diff
@@ -66,6 +66,11 @@ binary tree (*/path/to/ITKSphinxExamples-build/Utilities*)::
   cd /path/to/ITKSphinxExamples-build/Utilities
   python ./CreateNewExample.py
 
+The script will ask for an *example_name* and a *class_name*.  The example_name
+might be *CreateANewExample* for an example that answers the question "How do I
```
Comment (dzenanz) [inline Documentation/Contribute/ContributeWithGit.rst:70]: Too meta. Maybe `CreateANewExample` -> `ComputeTumorIntensityVariation`, and `itk::Statistics::MaskedImageToHistogramFilter` for a class in focus.

**Review by dzenanz — APPROVED** [review]

File: Documentation/Contribute/ContributeWithGit.rst (hunk 1 above)
Comment (Leengit) [reply Documentation/Contribute/ContributeWithGit.rst]: Thank you.  Done.

**Review by dzenanz — APPROVED** [review]

**Comment by Leengit** [comment]
I just rebased this change onto the commit that fixes the unrelated test failure.

**Pull request merged**